src/api-docs.js

```diff
@@ -1175,6 +1175,215 @@ const apiDoc = {
+                    summary: 'Wallet is not ready yet',
+                    value: { success: false, message: 'Wallet is not ready.', state: 1 }
+                  },
+                  'no-wallet-id': {
+                    summary: 'No wallet id parameter',
+                    value: { success: false, message: "Parameter 'wallet-id' is required." }
+                  },
+                  'invalid-wallet-id': {
+                    summary: 'Wallet id parameter is invalid',
+                    value: { success: false, message: 'Invalid wallet-id parameter.' }
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+    '/wallet/p2sh/tx-proposal/mint-tokens': {
+      post: {
+        summary: 'Get the hex representation of a mint tokens transaction without input data.',
+        parameters: [
+          {
+            name: 'x-wallet-id',
+            in: 'header',
+            description: 'Define the key of the corresponding wallet it will be executed the request.',
+            required: true,
+            schema: {
+              type: 'string',
+            },
+          }
+        ],
+        requestBody: {
+          description: 'Data to mint tokens.',
+          required: true,
+          content: {
+            'application/json': {
+              schema: {
+                type: 'object',
+                required: ['token', 'amount'],
+                properties: {
+                  token: {
+                    type: 'string',
+                    description: 'UID of the token to mint.'
+                  },
+                  amount: {
+                    type: 'integer',
+                    description: 'The amount of tokens to mint. It must be an integer with the value in cents, i.e., 123 means 1.23.'
+                  },
+                  address: {
+                    type: 'string',
+                    description: 'Optional destination address of the minted tokens.'
+                  },
+                  change_address: {
+                    type: 'string',
+                    description: 'Optional address to send the change amount.'
+                  },
+                  create_mint: {
+                    type: 'boolean',
+                    description: 'If should create another mint authority for the created token. Default is true.'
+                  },
+                  mint_authority_address: {
+                    type: 'string',
+                    description: 'Optional address to send the new mint authority output created.'
+                  },
+                  allow_external_mint_authority_address: {
+                    type: 'boolean',
+                    description: 'If the mint authority address is allowed to be from another wallet. Default is false.'
+                  },
+                }
+              },
+              examples: {
+                data: {
+                  amount: 100,
+                }
```

Comment:
Missing token parameter on the example
```suggestion
                data: {
                  token: 'tokenUID',
                  amount: 100,
                }
```